sampleReact.jsx

```
@@ -0,0 +1,11 @@
+function TestComponent() {
+    // ruleid:react-dangerouslysetinnerhtml
+    return <div dangerouslySetInnerHTML={createMarkup()} />;
+}
+
+function OkComponent() {
```

ESLint

[error] 'OkComponent' is defined but never used.

(@typescript-eslint/no-unused-vars)